### ISSUE TITLE
Save Error Bug?

### ISSUE THREAD

**Issue by Ketsueki-Kitsune** [KyonkoYuuki/YaMovesetOrganizer#9]
So I've made 2 successful movesets over the last few months, one personal before the patcher updated and one for a friend just recently.

However, now for whatever reason, when attempting to save a moveset, I get an error box telling me that there were problems with lines of code, and checking the save location only has the bac and bdm files saved successfully, but not the ean or bcm files like it did previously.

![bandicam 2024-08-20 16-05-02-626](https://github.com/user-attachments/assets/1f2b54ee-25b4-4971-9f60-7116c536d59f)

It's also followed by the aforementioned error box that I can't make heads or tails of which is worrying.

![bandicam 2024-08-20 06-21-45-937](https://github.com/user-attachments/assets/00a90a58-376e-490f-998e-6f3fd41d5ca5)

Any help would be appreciated.

**Comment by HectorLabrin** [comment]
yeah, that happen to me too, yesterday I test a moveset and save it and the next day the same error box show up

**Comment by Chaos-Blue** [comment]
Huh, yeah I'm also having this issue
